media-source-respec.html

```diff
@@ -51,28 +51,35 @@
 
       mseDefGroupName: "media-source",
       mseContributors: [
```

Comment:
A couple others whom I believe merit acknowledgement are:

- Dale Curtis (@dalecurtis) - he helped by advising, reviewing, and socializing feature work. In particular, he discovered internally at Google a strong use case that helped motivate my proposal and development of the `SourceBuffer.changeType()` method for seamless codec and bytestream format switching.
- Chris Wilson (@cwilso) - helped by assisting setup of the WICG media-source incubation after v1 was REC and before a WG chartered for v2 was available. `SourceBuffer.changeType()` and other early potential v2 feature incubations were made possible with his help and advice.

Reply:
Excellent, thanks, I have added both.